src/config.ts

```diff
@@ -78,21 +72,7 @@ export const baseDirPath = path.resolve(path.dirname(require.main.filename), '..
 /** Absolute path to the directory in which to put the jar files. */
 export const libDirPath = path.resolve(baseDirPath, libDirName);
 
-/** Absolute path to the directory in which to find the SWT jar files. */
-export const swtJarsDirPath = path.resolve(baseDirPath,
-    path.join(repoName, pkgName, 'target', 'repository', 'plugins'));
-
 /** Absolute path to the language and diagram server jar. */
-export const ldsJarFile = path.resolve(baseDirPath,
-    path.join(repoName, pkgName, 'target', 'exe', ldsJarName));
-
-/** Dictionary mapping OSes to the names of their corresponding SWT jars. */
-export const swtJarsByOs = defaultDict('org.eclipse.swt.gtk.linux.jar')({
-    'win32': 'org.eclipse.swt.win32.win32.jar',
-    'darwin': 'org.eclipse.swt.cocoa.macosx.jar'
-});
-
-/** Dictionary mapping OSes to their corresponding classpath separators. */
-export const classPathSeparatorsByOs = defaultDict(':')({
-    'win32': ';',
-});
+// TODO handle version in file name
+export const sourceLdsJarFile = path.resolve(baseDirPath,
+    path.join(repoName, 'org.lflang.diagram', 'build', 'libs', 'org.lflang.diagram-0.3.1-SNAPSHOT-lds.jar'));
```

Comment:
We probably shouldn't be hard coding the version number here.

Reply:
Yep, that's why I added the todo.
I suppose we can configure the shadow jar job in Gradle such that it does not put the version number into the file name or we need a dedicated field in the `config.ts`.
Could you take a look at this? Because I would like to continue the restructuring of the main repository as soon as https://github.com/lf-lang/lingua-franca/pull/1469 is merged.

Reply:
Could we just use a `*`? Also, for proper lease versions there will be no `SNAPSHOT` in the file name, so putting that there will likely also cause problems...

Reply:
I've created #86. So let's merge and fix later.